source/backend/api/Areas/Documents/DocumentRelationshipController.cs

```diff
@@ -62,45 +65,45 @@ public class DocumentRelationshipController : ControllerBase
         /// <param name="relationshipType">Used to identify document type.</param>
         /// <param name="parentId">Used to identify document's parent entity.</param>
         /// <returns></returns>
-        [HttpGet("{relationshipType}/{parentId:long}")]
+        [HttpGet("{relationshipType}/{parentId}")]
         [Produces("application/json")]
         [HasPermission(Permissions.DocumentView)]
         [ProducesResponseType(typeof(IList<DocumentRelationshipModel>), 200)]
         [SwaggerOperation(Tags = new[] { "document" })]
-        public IActionResult GetRelationshipDocuments(DocumentRelationType relationshipType, long parentId)
+        public IActionResult GetRelationshipDocuments(DocumentRelationType relationshipType, string parentId)
```

Comment:
huh, I don't really get this. Why is this better?

Reply:
Sadly, It is necessary. The id that the relationship has is a string (Form Type Code). Having the relationship use a string type allows both ids' as numbers and the type code as a string